packages/block-library/src/tag-cloud/style.scss

```diff
@@ -0,0 +1,10 @@
+.wp-block-tag-cloud {
+	&.aligncenter {
+		text-align: center;
+	}
+
+	&.alignfull {
+		padding-left: var(--global--spacing-unit, 20px);
```

Comment:
this doesn't sound like a Gutenberg CSS variable at all. Can we just remove it?

Reply:
Yeah, I believe it's coming from the twentytwentyone theme, also related to this issue https://github.com/WordPress/gutenberg/issues/27315.
Should definitely be removed here... :+1:

Reply:
Also noting that we should avoid using `px` values. We went to a lot of trouble converting everything to `em` units recently, we should not add new ones unless absolutely necessary.

Reply:
Thanks, I've fixed this.